fuzzers/035-iob-ilogic/generate.py

```diff
@@ -158,31 +178,25 @@ def main():
                 segmk.add_site_tag(site, 'IDELMUXE3.1', 1)
                 segmk.add_site_tag(site, 'IDELMUXE3.2', 0)
 
-                if ofb_used:
-                    segmk.add_site_tag(site, 'IMUX.1', 1)
-                    segmk.add_site_tag(site, 'IMUX.0', 0)
-                else:
-                    segmk.add_site_tag(site, 'IMUX.1', 0)
-                    segmk.add_site_tag(site, 'IMUX.0', 1)
             elif d['mux_config'] == 'idelay':
                 segmk.add_site_tag(site, 'IDELMUXE3.0', 1)
                 segmk.add_site_tag(site, 'IDELMUXE3.1', 0)
                 segmk.add_site_tag(site, 'IDELMUXE3.2', 0)
 
-                if ofb_used:
-                    segmk.add_site_tag(site, 'IMUX.1', 1)
-                    segmk.add_site_tag(site, 'IMUX.0', 0)
-                else:
-                    segmk.add_site_tag(site, 'IMUX.1', 0)
-                    segmk.add_site_tag(site, 'IMUX.0', 1)
             elif d['mux_config'] == 'none':
                 segmk.add_site_tag(site, 'IDELMUXE3.0', 0)
                 segmk.add_site_tag(site, 'IDELMUXE3.1', 0)
                 segmk.add_site_tag(site, 'IDELMUXE3.2', 0)
             else:
                 assert False, d['mux_config']
 
-    segmk.compile()
+            if DEBUG_FUZZER:
+                for k in d:
+                    segmk.add_site_tag(
+                        site, 'param_' + k + '_' + str(d[k]).replace(
+                            ' ', '').replace('\n', ''), 1)
+
+    segmk.compile(bitfilter=bitfilter)
     segmk.write(allow_empty=True)
```

Comment:
Looking into the code `allow_empty` lifts the assertion when there are no segment bits for a tag. Why is it needed here?

Reply:
It is possible that some specimens will not generate any segbits, but otherwise has no affect.  Solution should be returned.